application/src/test/java/uk/gov/hmcts/net/ccd/definition/store/rest/BaseTypeControllerTests.java

```diff
@@ -39,86 +40,31 @@ public void getBaseTypesTest() throws Exception {
-        assertNull("Unexpected Reg Ex", baseTypes[9].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "Complex", baseTypes[10].getType());
-        assertEquals("Unexpected Base Type Id", "Complex", baseTypes[10].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[10].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "Collection", baseTypes[11].getType());
-        assertEquals("Unexpected Base Type Id", "Collection", baseTypes[11].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[11].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "MultiSelectList", baseTypes[12].getType());
-        assertEquals("Unexpected Base Type Id", "MultiSelectList", baseTypes[12].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[12].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "Document", baseTypes[13].getType());
-        assertEquals("Unexpected Base Type Id", "Document", baseTypes[13].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[13].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "Label", baseTypes[14].getType());
-        assertEquals("Unexpected Base Type Id", "Label", baseTypes[14].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[14].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "DateTime", baseTypes[15].getType());
-        assertEquals("Unexpected Base Type Id", "DateTime", baseTypes[15].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[15].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "CasePaymentHistoryViewer", baseTypes[16].getType());
-        assertEquals("Unexpected Base Type Id", "CasePaymentHistoryViewer", baseTypes[16].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[16].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "FixedRadioList", baseTypes[17].getType());
-        assertEquals("Unexpected Base Type Id", "FixedRadioList", baseTypes[17].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[17].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "CaseHistoryViewer", baseTypes[18].getType());
-        assertEquals("Unexpected Base Type Id", "CaseHistoryViewer", baseTypes[18].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[18].getRegularExpression());
-
-        assertEquals("Unexpected Base Type", "DynamicList", baseTypes[19].getType());
-        assertEquals("Unexpected Base Type Id", "DynamicList", baseTypes[19].getId());
-        assertNull("Unexpected Reg Ex", baseTypes[19].getRegularExpression());
-
+        assertThat(baseTypes)
+            .withFailMessage("Unexpected Base Type and Id")
+            .extracting(FieldType::getId, FieldType::getType, FieldType::getRegularExpression)
+            .containsExactlyInAnyOrder(
+                tuple("Text", "Text", null),
```

Comment:
are the 3rd `null` parameters required since it takes variadic args?

Reply:
correct but just wan to show regex is null explicitly for some fields